samples/drivers/led_lp50xx/README.rst

```diff
@@ -30,9 +30,9 @@ For each color in red green blue white yellow purple cyan orange:
 Building and Running
 ********************
 
-This sample can be built and executed on boards with an I2C LP5030/6 LED driver
-connected. A node matching the "ti,lp503x" binding should be defined in the
-board DTS files.
+This sample can be built and executed on boards with an I2C LP5030/6 or
+LP50{09/12} LED driver connected. A node matching the
+ie. "ti,lp50xx", "ti,lp5030" binding should be defined in the board DTS files.
```

Comment:
Suggestion:

`A node matching both the "ti,lp50xx" binding and the device type should be defined in the board DTS files.`

As there is an example given right after, it should be good enough.